terraform/env_types/gcp/gke/gke_create/variables.tf

```diff
@@ -64,6 +64,10 @@ variable "pods_cidr" {
   description = "GKE pods CIDR"
 }
 
+variable "service_cidr" {
+  description = "GKE service CIDR"
+}
```

Comment:
Maybe we should add a default value?

Reply:
ideally - yes, practically - it has to be part of cluster VPC subnet which is unknown for this module.